app/controllers/media/objects_controller.rb

```diff
@@ -4,11 +4,17 @@ class Media::ObjectsController < ApplicationController
 
   def show
     style = params[:style]
-    send_file @media_object.item.path(style), type: @media_object.item_content_type, disposition: "inline"
+    @answer = @media_object.answer
```

Comment:
I wasn't sure if authorization was part of another sub-issue, or its own issue, I figured that we'd be doing something with authorizing access to media files though, otherwise there would be little reason not to serve them from accessible folders. I can do that here though.